docs/releases.rst

```diff
@@ -5,6 +5,13 @@
 ======================
 
 
+tmt-1.47.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The console log content is now available for guests provisioned by
```

Comment:
@psss would it make sense to document somewhere in docs, that the console log is saved and where the user should look for it?

Reply:
Yeah, good point, will do.